### PR TITLE
SingularityNET - Strategies

### PR DESCRIPTION
**Pull request by Vivek205**
Changes proposed in this pull request:
- *snet-liquidity-providers* : This strategy returns the liquidity pool balances of the voters for the SingularityNET project.
- *snet-stakers* : This strategy returns the staking pool balances of the voters for the SingularityNET project.
- *snet-farmers* : This strategy returns the yield-farming pool balances of the voters for the SingularityNET project.

### CODE REVIEW

File: src/strategies/index.ts [hunk 1]
```diff
@@ -395,7 +398,10 @@ const strategies = {
   'push-voting-power': pushVotingPower,
   'staked-psp-balance': stakedPSPBalance,
   'erc20-balance-of-contract-multiplier': erc20BalanceOfContractMultiplier,
-  agave
+  agave,
+  'snet-liquidity-providers': snetLiquidityProviders,
+  'snet-stakers': snetStakers,
+  'snet-farmers': snetFarmers
```
Comment (ChaituVR) [inline src/strategies/index.ts:404]: Hi @Vivek205 Would be great if you can do separate PR for each of these strategies, will be easy to review/integrate

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/index.ts (hunk 1 above)
Comment (Vivek205) [reply src/strategies/index.ts:404]: Sure. Will update it.

Comment (Vivek205) [reply src/strategies/index.ts:404]: @ChaituVR 

raised a separate PR for snet-liquidity-providers
https://github.com/snapshot-labs/snapshot-strategies/pull/202

Comment (Vivek205) [reply src/strategies/index.ts:404]: https://github.com/snapshot-labs/snapshot-strategies/pull/203

https://github.com/snapshot-labs/snapshot-strategies/pull/204
